### ISSUE TITLE
Create app from github always need repository permissions

### ISSUE THREAD

**Issue by wangcode** [ledokku/ledokku#404]
# Bug report

## Describe the bug

always need me to set repository permissions when create a app from my github repository.

## System information

- Version of dokku: [e.g. 0.25.1]
- Version of ledokku: [e.g. 0.7.0]

**Comment by wangcode** [comment]
![image](https://user-images.githubusercontent.com/10222039/129862486-30c89dcc-0186-428f-b4ef-b63bce54caec.png)

**Comment by Akirtovskis** [comment]
@wangcode Thanks for opening the issue. Which dokku and ledokku versions are you using? Was there time between two tries? Could you describe the issue in more detail so I can get more context and try to replicate it?

**Comment by wangcode** [comment]
@Akirtovskis My dokku version is 0.25.1 and the ledokku version is the lastest(0.7.0), I got this message ever tried,more then two times, I tried again just now, i use this just normal way, 
1. open new window to set github
2. give ledokku all repository permissions
3. close the new window
4. reload the old tab and goto the homepage
5. create app with github
6. got message

it's this has any way to show the nodejs logs?
